### PR TITLE
Release v4.1.0

### PR DESCRIPTION
**Pull request by Akarshit**
# v4.1.0
This release is coordinated with the release of [reaction-admin v4.0.0-beta.14](https://github.com/reactioncommerce/reaction-admin/releases/tag/v4.0.0-beta.14), [reaction v4.1.5](https://github.com/reactioncommerce/reaction/releases/tag/v4.1.5), [example-storefront v5.1.1](https://github.com/reactioncommerce/example-storefront/releases/tag/v5.1.1) to keep the  up-to-date with the latest version of all our development platform projects.

### CODE REVIEW

**Review by manueldelreal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -258,9 +258,9 @@ The following table provides the most current version of each project used by th
 | Project                             | Latest release / tag                                                                     |
 | ----------------------------------- | ---------------------------------------------------------------------------------------- |
 | [reaction-development-platform][10] | [`4.0.1`](https://github.com/reactioncommerce/reaction-development-platform/tree/v4.0.1) |
-| [reaction][10]                      | [`4.0.0`](https://github.com/reactioncommerce/reaction/tree/v4.0.0)                      |
-| [example-storefront][13]            | [`5.0.3`](https://github.com/reactioncommerce/example-storefront/tree/v5.0.3)            |
-| [reaction-admin (beta)][19]         | [`4.0.0-beta.5`](https://github.com/reactioncommerce/reaction-admin/tree/v4.0.0-beta.5)  |
+| [reaction][10]                      | [`4.1.5`](https://github.com/reactioncommerce/reaction/tree/v4.1.5)                      |
+| [example-storefront][13]            | [`5.1.1`](https://github.com/reactioncommerce/example-storefront/tree/v5.1.1)            |
+| [reaction-admin (beta)][19]         | [`4.0.0-beta.14`](https://github.com/reactioncommerce/reaction-admin/tree/v4.0.0-beta.14)  |
```
Comment (zenweasel) [inline README.md:263]: Is this supposed to say (beta)[19] or [14]?

**Comment by brent-hoover** [comment]
I tested this using these versions:

1. The admin does not load. Getting this error:
```
Uncaught Error: Cannot find module '@material-ui/core/styles/createMuiTheme'
    at y (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at d.n [as require] (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at e (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at e (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:11)
    at g (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at d.n [as require] (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at d.o [as link] (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:11)
    at e (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:11)
    at g (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
    at d.n [as require] (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:9)
f2af18cd66a5eb631f4c498002c933e73236c7d4.js?meteor_js_resource=true:1018 Uncaught TypeError: Cannot destructure property 'ServiceConfiguration' of 'Package.service-configuration' as it is undefined.
    at f2af18cd66a5eb631f4c498002c933e73236c7d4.js:1018
    at a (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:1)
    at HTMLDocument.u (f2af18cd66a5eb631f4c498002c933e73236c7d4.js:1)
```

Also if I revert back to the previous admin and attempt to checkout, even with payments enabled I get:

image

image

Seeing this error in the browser console:
```
 [GraphQL error]: Variable "$shopId" of non-null type "ID!" must not be null. {locations: Array(1), operationName: 'availablePaymentMethods', path: undefined}
```

**Comment by delagroove** [comment]
Also can we move this release number to be 4.0.2 instead? there's real reason why we are jumping a minor version in the release

**Comment by Akarshit** [comment]
@delagroove Both `reaction` and `example-storefront` have been bumped up by a minor version. Hence the minor version update.

**Pull request closed without merging**